tests/e2e/__testers__.ts

```diff
@@ -4,29 +4,48 @@ import { values } from 'lodash'
 import stripAnsi from 'strip-ansi'
 import { ClientBase } from '@prisma-spectrum/reflector/dist-cjs/Client'
 import { Reflector } from '@prisma-spectrum/reflector'
+import { getPrismaClient } from '~/tests/e2e/helpers/getPostgresAdminPrismaClient'
+import { PrismaClient } from '@prisma/client'
 
-export const testTemplate = (params: {
+export interface DBTestParams {
   templateName: PrismaTemplates.$Types.Template['_tag']
-  /** A string or regex pattern of the expected output (stdout) when the project dev mode is run.*/
   expectedDevOutput: RegExp | string
-}) => {
-  jest.setTimeout(50_000)
+  testDBURI?: string
+  datasourceProvider?: Reflector.Schema.DatasourceProviderNormalized
+  databaseUrlBase: string
```

Comment:
Can `databaseUrlBase` and `testDBURI` be aligned? 

`connectionStringBase` and `connectionStringApp`?

What is `connectionStringBase` for? If all databases are setup with same user/password credentials then we should be able to automatically figure this out, no config needed.

Reply:
`testDBURI` was a left over, nice catch.

I'll rename `databaseUrlBase` to `connectionStringBase`.

The config is necessary to avoid hardcoding inside a function which is widely used.